mwl_phonemizer/crf_mwl.py

```
@@ -1,82 +1,239 @@
         super().__init__(*args, **kwargs)
         self.crf_model_path = crf_model_path
+        self.algorithm = algorithm
+        self.c1 = c1
+        self.c2 = c2
+        self.max_iterations = max_iterations
+        self.all_possible_transitions = all_possible_transitions
+        self.strategy = strategy
+        self.manual_fixes = apply_manual_fixes
         self.model = None
+        self.ignore_stress = ignore_stress
         if crf_model_path and os.path.exists(crf_model_path):
```

Ruff (0.13.2)

Undefined name `os`

(F821)